Modules/_uuidmodule.c

```diff
@@ -91,45 +111,1649 @@ py_windows_has_stable_node(void)
 #endif /* MS_WINDOWS */
 
 
-static int
-uuid_exec(PyObject *module)
-{
-#define ADD_INT(NAME, VALUE)                                        \
-    do {                                                            \
-        if (PyModule_AddIntConstant(module, (NAME), (VALUE)) < 0) { \
-           return -1;                                               \
-        }                                                           \
-    } while (0)
+typedef struct uuidobject {
+    PyObject_HEAD
+    uint8_t bytes[16];
+    Py_hash_t cached_hash;
+    PyObject *is_safe;
+    PyObject *weakreflist;
+} uuidobject;
 
-    assert(sizeof(uuid_t) == 16);
-#if defined(MS_WINDOWS)
-    ADD_INT("has_uuid_generate_time_safe", 0);
```

Comment:
Where did this go?

Reply:
I don't see this in my diff, perhaps you were reviewing an older version. This code of course stays and isn't touched by me.